model/storage-private/src/main/java/org/keycloak/storage/datastore/DefaultDatastoreProviderFactory.java

```diff
@@ -99,34 +101,45 @@ public void onEvent(ProviderEvent event) {
             StoreMigrateRepresentationEvent ev = (StoreMigrateRepresentationEvent) event;
             MigrationModelManager.migrateImport(ev.getSession(), ev.getRealm(), ev.getRep(), ev.isSkipUserDependent());
         }
-    }    
-
-    public void setupScheduledTasks(final KeycloakSessionFactory sessionFactory) {
-        long interval = Config.scope("scheduled").getLong("interval", 900L) * 1000;
+    }
 
+    public static void setupScheduledTasks(final KeycloakSessionFactory sessionFactory) {
         try (KeycloakSession session = sessionFactory.create()) {
             TimerProvider timer = session.getProvider(TimerProvider.class);
             if (timer != null) {
-                scheduleTasks(sessionFactory, timer, interval);
+                scheduleTasks(sessionFactory, timer, getScheduledInterval());
             }
         }
     }
 
-    protected void scheduleTasks(KeycloakSessionFactory sessionFactory, TimerProvider timer, long interval) {
+    protected static void scheduleTasks(KeycloakSessionFactory sessionFactory, TimerProvider timer, long interval) {
         for (ScheduledTask task : getScheduledTasks()) {
             scheduleTask(timer, sessionFactory, task, interval);
         }
 
         UserStorageSyncManager.bootstrapPeriodic(sessionFactory, timer);
     }
 
-    protected List<ScheduledTask> getScheduledTasks() {
+    protected static List<ScheduledTask> getScheduledTasks() {
         return Arrays.asList(new ClearExpiredEvents(), new ClearExpiredAdminEvents(), new ClearExpiredClientInitialAccessTokens(), new ClearExpiredUserSessions());
     }
 
-    protected void scheduleTask(TimerProvider timer, KeycloakSessionFactory sessionFactory, ScheduledTask task, long interval) {
+    protected static void scheduleTask(TimerProvider timer, KeycloakSessionFactory sessionFactory, ScheduledTask task, long interval) {
         timer.schedule(new ClusterAwareScheduledTaskRunner(sessionFactory, task, interval), interval);
         logger.debugf("Scheduled cluster task %s with interval %s ms", task.getTaskName(), interval);
     }
 
+    public static void setupCleanupExpiredRevokeTokenScheduledTask(KeycloakSessionFactory sessionFactory) {
```

Comment:
Nitpick: The name of the method should be aligned more with the task `ClearExpiredRevokedTokens`:

```suggestion
    public static void setupClearExpiredRevokedTokensScheduledTask(KeycloakSessionFactory sessionFactory) {
```